### PR TITLE
Updated gitignore, added documentation

### PR DESCRIPTION
**Pull request by Boundarybreaker**
Filled out the files for "what is correlated multitasking" and "modules", giving modules a 403 since it's NYI

### CODE REVIEW

File: src/main/resources/documentation/en_US/global/whatcom.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+##But what does this have to do with storage?
```
Comment (unascribed) [inline src/main/resources/documentation/en_US/global/whatcom.md]: There's a lot of "However"s in this section

File: src/main/resources/documentation/en_US/global/whatcom.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# What is Correlated Multitasking?
+
+Correlated Multitasking is newest incredible development from Correlated Logistics ltd, tapping into the incredible power of the Enderspace!
```
Comment (unascribed) [inline src/main/resources/documentation/en_US/global/whatcom.md]: This should probably be "incredible power of Ender" - Ender has more properties that just accessing enderspace

File: src/main/resources/documentation/en_US/global/whatcom.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+##The Enderspace Race
+There is a common misconception that digital storage converts items into data and annihilates them, to be reformed later based on the saved data. While some have attempted this, the energy costs involved are astronomical, and if it were implemented, you could create any matter out of pure energy, which violates all conventions of physics. No, Correlated Logistics uses a revolutionary material called Ender, which allows transport of matter or energy to and from "Enderspace" — a seemingly infinite in-between space generally used for teleportation by Endermen. Humans harness this fantastical power whenever they throw an ender pearl - For reasons heretofore unknown, shattering a pearl warps the caster to the location the pearl shatters at. At last, the mystery has been solved: the shattered pearl releases its Ender energy, causing the one who caused its shattering to hurtle through Enderspace to the point at which it shattered. Humans are not able to survive in enderspace for long, so this process inevitably causes some harm. Correlated Logistics is actively working on protective methods for humans to one day traverse this space themselves.
```
Comment (unascribed) [inline src/main/resources/documentation/en_US/global/whatcom.md]: Also, this entire file should get folded to 80 columns to match the other files. Not all editors do soft wrapping.

On Linux, you can do this pretty easily; `mv file file.unfolded; fold -s file.unfolded > file`

Comment (Boundarybreaker) [reply src/main/resources/documentation/en_US/global/whatcom.md]: Folded with new push!

File: src/main/resources/documentation/en_US/global/whatcom.md.unfolded [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# What is Correlated
```
Comment (unascribed) [inline src/main/resources/documentation/en_US/global/whatcom.md.unfolded]: This file should probably be deleted

File: src/main/java/com/elytradev/correlated/proxy/ClientProxy.java [hunk 5]
```diff
@@ -1108,17 +1108,20 @@ public void drawSprite(TextureAtlasSprite tas) {
...
 	public void onRenderTickForDebugOutput(RenderTickEvent e) {
 		if (e.phase == Phase.START && (Boolean)Launch.blackboard.get("fml.deobfuscatedEnvironment")) {
 			if (!checkedDebugSupport) {
-				if (!GLContext.getCapabilities().GL_KHR_debug) {
-					CLog.info("Your driver doesn't support KHR_debug_output...");
-					return;
-				} else {
-					CLog.info("KHR_debug_output is supported. Enabling OpenGL debug output!");
-				}
+					if (!GLContext.getCapabilities().GL_KHR_debug) {
```
Comment (unascribed) [inline src/main/java/com/elytradev/correlated/proxy/ClientProxy.java]: It looks like this is indented one level too far

**Pull request merged**